dev-requirements.txt

```diff
@@ -7,7 +7,8 @@ tornado==5.0.2
 PySocks==1.6.8
 pkginfo==1.4.2
 pytest-timeout==1.3.1
-pytest==3.6.4
+pytest==3.8.2
 # https://github.com/GoogleCloudPlatform/python-repo-tools/issues/23
 pylint<2.0;python_version<="2.7"
 gcp-devrel-py-tools==0.0.15
+pluggy==0.8.0
```

Comment:
```suggestion
```

Reply:
Can we get away with not pinning pluggy here?

Reply:
We can get away with not pinning pluggy, but keep in mind that a new version of pluggy is what broke the build yesterday.  I saw the other requirements that were pinned to specific versions and inferred that this project likes to have stable dependencies but the build will succeed without it.

Reply:
The commit that upgrades pytest (and fixes the build) is 41a665a and the one that pins pluggy (which is just future-proofing) is 2f21ef8 so if you *don't* want to pin pluggy you can merge 41a665a but if you *do* you can merge 2f21ef8.

Reply:
Arguably that was a bug in pytest 3.6.4, which should have pinned pluggy, just like requests pins urllib3. I think we either want to go the full pip-compile + dependabot route, or just list the dependencies we use, which means we'll get occasional breakage like this. Of course, the maintainers decide. :)

Reply:
Pytest 3.6.4 did require a range for pluggy (>=0.5,<0.8) but some other module that was loaded sooner caused pluggy 0.8.0 to load so when pytest asked for <0.8 things got crashy.  The pain of "DLL hell" is still real.

Reply:
Oh, right. tox 2.9.1 requires `pluggy>=0.3.0,<1.0` and is placed before pytest, which is why pluggy 0.8.0 was installed. Really, the only real way to fix this is to use pip-compile, and possibly dependabot to help with the constant updates.

Reply:
I think merging this as-is is fine for now. We'll have to revisit our whole reproducible build strategy collectively to solve this problem completely :)